pandas/io/parsers.py

```diff
@@ -245,9 +245,9 @@
     Indicates remainder of line should not be parsed. If found at the beginning
     of a line, the line will be ignored altogether. This parameter must be a
     single character. Like empty lines (as long as ``skip_blank_lines=True``),
-    fully commented lines are ignored by the parameter `header` but not by
-    `skiprows`. For example, if comment='#', parsing '#empty\\na,b,c\\n1,2,3'
-    with `header=0` will result in 'a,b,c' being
+    fully commented lines are ignored by the parameter ``header`` but not by
```

Comment:
Can you leave 'header' (and skiprows below) with single backticks? (that's how literal references to kwargs are done in the rest of the docstring I think). 
For the code example, the double ones are good!

Reply:
yup, done